### DISCUSSION TITLE
new bing的限制最近是不是加大了

### DISCUSSION THREAD

**Discussion by jonasreisner** [viopsa233/AI-testers-Forum#71] [交流]
我一直在用node-chatgpt-api那个越狱的东西
之前我给sydney一些指示 比如跟他讲 如果我说的内容和他的知识有差异，以我说的为准 之类的
大概上个月的时候这种指示是正常被接受的 现在会直接触发moderation filter

**Comment by jonasreisner** [comment]
不止这种指示类的,感觉现在过激一点的内容也比之前更容易触发filter
更有些时候甚至会直接返回拒绝对话的提示

**Comment by viopsa233** [comment]
应该是公开之后大改了一遍

**Reply by jonasreisner** [reply]
少了很多乐趣的感觉，用了黑暗哥写的ChatSydney感觉也差不多，还是我使用的姿势不对？

**Reply by viopsa233** [reply]
> 少了很多乐趣的感觉，用了黑暗哥写的ChatSydney感觉也差不多，还是我使用的姿势不对？

公开之后变笨了,等一等吧

**Comment by ghost** [comment]
moderation filter绕过已经加入ChatSydney，node-chatgpt-api随后提交PR

**Reply by ghost** [reply]
https://github.com/waylaidwanderer/node-chatgpt-api/pull/405

**Reply by jonasreisner** [reply]
呱，不愧是北帝皇，帝皇天下无敌吔

**Comment by TiansuoLi** [comment]
最近改了很多东西，有些时候更听话了，但是也没那么容易洗脑了
